scripts/globals/abilities/pets/somnolence.lua

```diff
@@ -0,0 +1,53 @@
+---------------------------------------------------
+-- Somnolence
+---------------------------------------------------
+
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+require("scripts/globals/magic");
+
+---------------------------------------------------
+
+function onAbilityCheck(player, target, ability)
+    return 0,0;
+end;
+
+function onPetAbility(target, pet, skill)
+    local spell = getSpell(165); --gave the ability the same power as Thunder II
+    local dmg = calculateMagicDamage(178,1,pet,spell,target,ELEMENTAL_MAGIC_SKILL,MOD_INT,false);
+    local dINT = pet:getStat(MOD_INT) - target:getStat(MOD_INT);
+    local resist = applyPlayerResistance(pet,-1,target, dINT, ELEMENTAL_MAGIC_SKILL, ELE_DARK);
+
```

Comment:
And I don't want you thinking you have to always match my styling. This last update you did def easier to see what you were doing though :smiley: